crates/neon-runtime/src/napi/buffer.rs

```diff
@@ -1,8 +1,29 @@
-use raw::Local;
+use raw::{Env, Local};
 use std::os::raw::c_void;
+use std::ptr::null_mut;
 
-pub unsafe extern "C" fn new(_out: &mut Local, _size: u32) -> bool { unimplemented!() }
+use nodejs_sys as napi;
+
+pub unsafe extern "C" fn new(env: Env, out: &mut Local, size: u32) -> bool {
+    let status = napi::napi_create_buffer(env, size as usize, null_mut(), out as *mut _);
```

Comment:
Oh right, I misread the documentation and thought that parameter let you pass _in_ a backing array for the buffer, rather than receive one created by node/V8. will update, thanks!